std-bits/table/src/main/java/org/enso/table/data/column/operation/cast/CastProblemBuilder.java

```diff
@@ -0,0 +1,13 @@
+package org.enso.table.data.column.operation.cast;
+
+public class CastProblemBuilder {
+  private int failedConversionsCount = 0;
+
+  public void reportConversionFailure() {
+    failedConversionsCount++;
+  }
+
+  public int getFailedConversionsCount() {
```

Comment:
Is there a way for the user to know which values failed to convert?

Reply:
Currently, not directly.

They are replaced with `Nothing` in the resulting table (although there can be existing Nothing values in there, so it is not unambiguous).

Do you think there should be? I was afraid that the error message may get too long.

But maybe we can gather the first 3... or 10? values that failed to convert and display these. I guess that might be helpful. I can add it. Shall I?

Reply:
What do you think a decent limit would be? IMO 3 sounds good as it shows a few examples. Anything more will make the error message too long anyway.

Reply:
I'm not really sure; a few might be useful, because that might be enough to fix the problems. On the other hand, the user might have to iterate many times to find all the errors, if there are a lot of problems. And I would hope that at some point we have a way to implement this that doesn't require that we re-implement it for every Column method. :)

Reply:
I think this is fine for now, until we have a more general plan.

Reply:
> And I would hope that at some point we have a way to implement this that doesn't require that we re-implement it for every Column method. :)

I think error handling is very usage-specific, so there is no one-fits-all solution.

In fact, I'm leaning towards creating tailored ones for each method (group), as I first attempted to create a generic `Problem_Builder` but it turns out it gets unnecessarily complex if we try to solve all kinds of scenarios using one tool.